www/blog/2023/06/19/karmashapes/index.md

```diff
@@ -0,0 +1,790 @@
+
+Even with those earlier efforts, WOF heard consistent feedback from our users:
+
+1. _Some localities are missing entirely from the WOF gazetteer_
+2. _Some point localities are offset in WOF compared to their real ground position_
+3. _Some polygon localities in WOF have the wrong shape compared with their real ground position (too small, too large, too coarse, or otherwise misshapen)_
+4. _Existing locality data is sometimes miscategorized/misclassified and would be better represented as instead with a different WOF placetype of neighbourhood or borough_
+
+These data gaps create challenges when operating location service applications like search and reverse geocoding for India. When a place is missing, it can’t be searched by name using a partial or complete text string. And the absence of places, incorrect centroids, and incorrect polygons for existing WOF places can cause reverse geocoding services to [hallucinate](https://en.wikipedia.org/wiki/Hallucination_(artificial_intelligence)#:~:text=In%20artificial%20intelligence%20(AI)%2C,insufficient%2C%20biased%20or%20too%20specialised.) when answering the question of _“What locality is here?”_
+
+As anyone who’s searched for Indian map data knows, the sheer number of people and populated places, combined with the historic paucity of openly licensed and/or authoritative government data, has presented significant challenges. India recently [surpassed](https://www.un.org/development/desa/dpad/publication/un-desa-policy-brief-no-153-india-overtakes-china-as-the-worlds-most-populous-country/#:~:text=In%20April%202023%2C%20India's%20population,to%20grow%20for%20several%20decades.) China to become the most populous country on Earth with over 1.4 billion people; most are clustered in over 500,000 rural villages, yet some of the towns and urban areas have incredible density themselves.
```

Comment:
Since this section now starts with the 500k discussion, let's pivot this later graph to focus on the % rural statistic instead?

To: `Around 65% of people in India are spread out in hundreds of thousands of rural villages with populations less than 5,000; the remaining 35% urbanized population is concentrated in towns with incredible density themselves.`

From: `most are clustered in over 500,000 rural villages, yet some of the towns and urban areas have incredible density themselves.`

- https://data.worldbank.org/indicator/SP.URB.TOTL.IN.ZS?locations=IN